docs/edgekv_cli.md

```diff
@@ -152,6 +152,7 @@ Usage: `akamai edgekv create ns <environment> <nameSpace>`
 | -h, --help  | optional | Display information on how to use this EdgeKV command. |
 | --retention | Required | Retention period of the namespace in days. |
 | --groupId | Required | Group identifier. Set it to 0 to allow all groups in your account to access the namespace. If you want to restrict the namespace to a specific group, enter the group id. This value MUST be the same for both the staging and production instances of a namespace. |
+| --geolocation | Required | Specifies the storage location for data when creating a namespace on the production network. This can help optimize performance by storing data where most or all of your users are located. The value defaults to `US` on the `STAGING` and `PRODUCTION` networks. The available locations also include `EU` (Europe) and `JP` (Japan) for the `PRODUCTION` network. Once you create this value, you can't modify it.|
```

Comment:
I suggest using "Specifies the persistent storage location" to avoid any confusion that this limits everywhere the data is stored in the system. Please check with Sarah on this suggestion.

Reply:
Also please remove " The available locations also include `EU` (Europe) and `JP` (Japan) for the `PRODUCTION` network. Once you create this value, you can't modify it.|". We do not want to explicitly list the geos here since we will be adding geos as we go and we want to avoid making changes to the CLI (new cli release) every time we add a new geo. Please consult with Sarah on where to point the user to get the latest supported geos from our documentation.